src/main/java/pl/lodz/p/it/zzpj/botsite/web/controllers/MessageAdminController.java

```diff
@@ -90,11 +91,17 @@ public MessageDTO editMessageForUser(@PathVariable("userId") Long userId, @PathV
             consumes = MediaType.APPLICATION_JSON_VALUE
     )
     @ResponseStatus(HttpStatus.OK)
-    public void deleteMessageForUser(@PathVariable("userId") Long userId, @PathVariable("messageId") Long messageId) throws MessageRetrievalException, MessageDeletionException {
+    public void deleteMessageForUser(@PathVariable("userId") Long userId, @PathVariable("messageId") Long messageId) throws MessageRetrievalException, MessageDeletionException, MessageNotFoundException {
```

Comment:
Jaki przypadek użycia dla admina potrzebuje mieć metode delete for user? Pozbyj się tego, zaśmieca. Id jest unialne w message. Wystarczy, że admin zna id i tyle.